library/src/scripts/theming/ThemeChooserInput.tsx

```diff
@@ -65,7 +67,7 @@ export function ThemeChooserInput(props: IProps) {
                 value: option.value,
             };
         }
-    }) ?? { label: t("Unknown"), value: props.initialValue };
+    }) ?? { label: `Default (${defaultTheme.name})`, value: props.initialValue };
```

Comment:
Shouldn't you be using `<Translate` with a placeholder here?

Reply:
should be good now